sql-select/exercise/all-actors.sql

```diff
@@ -0,0 +1 @@
+select * from actors;
```

Comment:
Please make a habit of formatting your SQL queries on multiple lines so they're easier to read:

---

![image](https://user-images.githubusercontent.com/16143771/118751862-e70b4980-b816-11eb-958d-b99ed2aff324.png)